HACKING.md

```diff
@@ -0,0 +1,66 @@
+# Hacking on Origin Aggregated Logging
+
+## Building on OpenShift
+Choose the project you want to hold your logging infrastructure. It can be
+any project.
```

Comment:
"The project/namespace `logging` is typically used for this, and many of the defaults and other documentation use this."